src/components/ColorPicker/index.tsx

```diff
@@ -32,7 +32,7 @@ const ColorPicker = ({
 }: ColorPickerProps): JSX.Element => {
     const [currentColor, setCurrentColor] = useState(initialColor);
     const [debouncedColor] = useDebounce(currentColor, 250);
-    const isInitialRender = useRef(true);
+    const isUserInteraction = useRef(false);
```

Comment:
Now that other elements are interacting with color changing, this needed to be a little more robust so prevent re-renders from triggering this and making preview events into permanent selection changes.